src/k8s/pkg/k8sd/setup/kube_apiserver_test.go

```diff
@@ -116,6 +119,9 @@ func TestKubeAPIServer(t *testing.T) {
 			{key: "--tls-cipher-suites", expectedVal: apiserverTLSCipherSuites},
 			{key: "--tls-private-key-file", expectedVal: path.Join(s.Mock.KubernetesPKIDir, "apiserver.key")},
 			{key: "--etcd-servers", expectedVal: fmt.Sprintf("unix://%s", path.Join(s.Mock.K8sDqliteStateDir, "k8s-dqlite.sock"))},
+			{key: "--anonymous-auth", expectedVal: "false"},
+			{key: "--profiling", expectedVal: "false"},
+			{key: "--requests-timeout", expectedVal: "300s"},
```

Comment:
same here, keep these sorted, it's hard going over these lists if they get out of order